### ISSUE TITLE
Content preview for Cardgroup element

### ISSUE THREAD

**Issue by lisanoffke** [benjaminkott/bootstrap_package#899]
# Feature Request

Currently the backend shows the cardgroups as they are regular containers.
But the cardgroup element does not show any cardgroup items in the backend page view.

## Is your feature request related to a problem? Please describe

No, it is simply nice to have to optimize editorial workflows.
Right now, when I use several cardgroup elements on one page I have to guess in which cardgroup element I would like to change a cardgroup item.

## Describe the solution you'd like

When in page view mode, I see all cardgroup items in the cardgroup element.

## Describe alternatives you've considered

it stays the way it is.

**Comment by gilbertsoft** [comment]
Are you talking about the list module here?

**Comment by olivermeckel** [comment]
You mean the presentation in the backend? Like this:

![CardGroup-Backend](https://user-images.githubusercontent.com/9928953/83938661-c64d0e00-a7d6-11ea-83bd-1d0632f774ce.jpg)

**Comment by lisanoffke** [comment]
Exactly. In this example, "Kartengruppe" contains several individual elements (if I'm not mistaken). The question is: Can you show these individual elements instead of the term "Kartengruppe"? That would make editorial work easier - just a suggestion.

**Comment by gilbertsoft** [comment]
@lisanoffke you should use better names for your content elements which should solve your issue imho. And no, this is a core functionality which we can not change with the current implementation.
